### ISSUE TITLE
Is it providing horizontal scrollable radio buttons instead of vertical scroll?

### ISSUE THREAD

**Issue by iAj43** [VenkateshYadavP/PVRadioButton#2]
![Screenshot 2020-01-22 at 4 41 44 PM](https://user-images.githubusercontent.com/20102573/72889718-50b83800-3d36-11ea-8f73-4b4d21b426ed.png)

**Comment by VenkateshYadavP** [comment]
No, It currently supports only a vertical scroll. If you are facing any issue, please help me with the code snippet.
